src/PrestaShopBundle/Resources/views/Admin/layout.html.twig

```diff
@@ -22,21 +22,22 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
-{% extends(template_from_string(
-  getLegacyLayout(
-    app.request.attributes.get('_legacy_controller'),
-    layoutTitle is defined ? layoutTitle : '',
-    layoutHeaderToolbarBtn is defined ? layoutHeaderToolbarBtn : [],
-    layoutDisplayType is defined ? layoutDisplayType : '',
-    showContentHeader is defined ? showContentHeader : true,
-    headerTabContent is defined ? headerTabContent : '',
-    enableSidebar is defined ? enableSidebar : false,
-    help_link is defined ? help_link : '',
-    js_router_metadata(),
-    meta_title is defined ? meta_title : '',
-    use_regular_h1_structure is defined ? use_regular_h1_structure : true
-  )
-  )) %}
+{% extends use_new_layout is defined
```

Comment:
Why check the `base.html.twig` exists ? You are worried it disappears ? 🤔

Reply:
I'm not checking if `base.html.twig` exists, I'm using it if `use_new_layout` is defined and the old `template_from_string(getLegacyLayout())` if not :)